webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -1250,45 +1177,181 @@ class ResourcesV1R2RSpec extends R2RSpec {
                 }
 
                 assert(utf8str == "another simple text")
+            }
+        }
 
+        "create resources from an XML import" in {
+            val xmlImport =
+                s"""<?xml version="1.0" encoding="UTF-8"?>
+                   |<knoraXmlImport:resources xmlns="http://api.knora.org/ontology/biblio/xml-import/v1#"
+                   |    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   |    xsi:schemaLocation="http://api.knora.org/ontology/biblio/xml-import/v1# biblio.xsd"
+                   |    xmlns:biblio="http://api.knora.org/ontology/biblio/xml-import/v1#"
```

Comment:
So in short this means, in XML an element or attribute name has solely to be uniquely identified but not dereferenceable?

We absolutely want our Iris used in RDF to be dereferenceable, I totally agree.